src/vue/filters/formatDate.spec.js

```diff
@@ -0,0 +1,19 @@
+import { formatDate } from './formatDate'
+import { i18nOptions } from '@/i18n'
+import i18next from 'i18next'
+
+describe('formatDate filter test', () => {
+  beforeEach(() => {
+    i18next.init(i18nOptions)
+    sinon.restore()
+    sinon.useFakeTimers({
+      now: 1542968022000 // 23 Nov 2018
+    })
+  })
+
+  it('formats the date', () => {
+    expect(formatDate('2017-11-20T10:23:45Z'))
```

Comment:
in case of elements that other elements that already tested in other place (in case of filters that use 18next’s formatters, they’re already tested) you can just ensure that you call the needed method, for the current example: you should just ensure that you call `.t()` of `i18next` with valid params.